### ISSUE TITLE
Tribler does not open after nupgrade to 7.3.0

### ISSUE THREAD

**Issue by 3rdChimp** [Tribler/tribler#4788]
#### Please, fill all relevant items: ####

- [ ] I have read CONTRIBUTING.rst

I cannot find this file

- [ ] I have tried with the latest pre-release version and I still can reproduce the issue.

I found only one release in the download directory

##### Tribler version/branch+revision: #####

7.3.0

##### Operating system and version: #####

Debian GNU/Linux 9.4 (stretch)
Linux version 4.9.0-8-amd64

##### Steps to reproduce the behavior: #####

Start tribbler (CLI or menu)

##### Expected behavior: #####

Tribbler GUI oppens

##### Actual behavior: #####

A window appears. It says "loading" on white over black, with rotating gears
The gear rotate for many seconds
then the window closes

##### Relevant log file output: #####

Please tell me the path of the log

**Comment by 3rdChimp** [comment]
pip -V
pip 9.0.1 from /usr/lib/python2.7/dist-packages (python 2.7)

# pip install --user wheel
Requirement already satisfied: wheel in /usr/local/lib/python2.7/dist-packages
# pip install --user pony lz4
Requirement already satisfied: pony in /usr/local/lib/python2.7/dist-packages
Requirement already satisfied: lz4 in /usr/local/lib/python2.7/dist-packages
Requirement already satisfied: future in /usr/local/lib/python2.7/dist-packages (from lz4)
#

**Comment by xoriole** [comment]
@3rdChimp Found that Debian 9 has older version of python cryptography (1.7.1-3+deb9u1 < 2.3 required) which could the reason for Tribler is failing to run on your machine. Could you try the following pip package and try again?

```
pip install --user aead
```

**Comment by 3rdChimp** [comment]
Same issue after installing aead

Shall I try and reinstall?
Where is the log?

**Comment by 3rdChimp** [comment]
Hi guys
Any update?

**Comment by xoriole** [comment]
@3rdChimp Logs are present in the following directory:
```
~/.Tribler/logs
```
The following files might be present there. Please have a look at them in the listed order for any stack traces and report back.


- crash-report.log
- tribler-gui-error.log
- tribler-core-error.log
- tribler-gui-info.log
- tribler-core-info.log

It could still be cryptography issue. Could you try the following?
```
sudo apt remove python-cryptography
pip uninstall cryptography
pip uninstall aead
pip install --user cryptography
```
This should update the cryptography module to the latest version (2.7).
Let us know if that helps, otherwise, there could be something useful in the logs for us.

**Comment by 3rdChimp** [comment]
Thanks.
There was nothing relevant in the logs
I did the procedure
and after that, the tribler was not even starting, like it was uninstalled, but the files were still here
I tried to reinstall the debian package:

sudo dpkg --install tribler_*_all.deb

but the system said it was missing dependancies python-cryptography and python-twisted

I tried to install :

sudo apt-get install python-cryptography

but apt-get said it was broken

sudo apt --fix-broken install

it repaired all

the tribler started. It spend one hour converting files, then it started for good and it seems ok now

Thanks guys. You are the best
